doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+.. _setup_logical_partition_scenario_on_kbl_nuc_i7:
+
+Guidelines of Setup Logical Partition Scenario on KBL NUC i7
```

Comment:
Any reason this would only work on Core i7 processors?

If not, let's be more generic throughout the doc and simply indicate that these steps were validated on the NUC you used.

Reply:
@gvancuts for the current vm configuration of logical partition, the physical board should have 4 physical cores, core i7 is selected.